src/editors/playgroundController.ts

```diff
@@ -316,13 +317,38 @@ export default class PlaygroundController {
     return this._createPlaygroundFileWithContent(content);
   }
 
+  async createPlaygroundFromTreeView(
+    treeItem: DatabaseTreeItem | CollectionTreeItem
+  ): Promise<boolean> {
+    let content = '';
+    if (treeItem instanceof DatabaseTreeItem) {
+      content = playgroundInsertDocumentTemplate.replace(
+        'CURRENT_DATABASE',
+        treeItem.databaseName
+      );
+      this._telemetryService.trackPlaygroundCreated('fromDatabaseTreeItem');
+    } else if (treeItem instanceof CollectionTreeItem) {
+      content = playgroundInsertDocumentTemplate
+        .replace('CURRENT_DATABASE', treeItem.databaseName)
+        .replace('CURRENT_COLLECTION', treeItem.collectionName);
```

Comment:
Keep in mind that both database and collection names can contain special characters, such as newlines and quotes 🙃

Reply:
Ah good point (and wow a database name can contain a newline...), though I guess we haven't been mindful about this generally. @alenakhineika thoughts on whether we'd want to handle this now or maybe have some ticket to keep track of special cases for corner case database/collection names?

Reply:
I'd handle it, if at all possible. We can use `JSON.stringify()` here, since these are just strings, that takes care of escaping as necessary.